### PR TITLE
Set up app insights event logging on the web

### PR DESCRIPTION
**Pull request by oreHGA**
- Install app insights.js package
- Loads a `appInsight.` object that can be called to log events

### CODE REVIEW

**Review by TemitopeAsama — APPROVED** [review]

File: frontend/src/pages/profile.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ const AccountPage: NextPage = () => {
   const [confirmDeleteAccount, setConfirmDeleteAccount] = React.useState(false);
 
   const deleteAccount = () => {
+    appInsights.trackEvent({ name: 'customEventName', properties: { customProperty: 'value' } });
```
Comment (oreHGA) [inline frontend/src/pages/profile.tsx:18]: let's use name: 'delete_account'

File: frontend/src/pages/auth/login.tsx [hunk 2]
```diff
@@ -49,15 +50,17 @@ const LoginPage = React.memo(() => {
   };
 
   const useExistingAccount = async (privateKey: string) => {
+    appInsights.trackEvent({ name: 'customEventName', properties: { customProperty: 'value' } });
```
Comment (oreHGA) [inline frontend/src/pages/auth/login.tsx:53]: let's update name: `use_existing_account`

File: frontend/src/utils/appInsights.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    enableCorsCorrelation: true,
+    enableRequestHeaderTracking: true,
+    enableResponseHeaderTracking: true
+  },
+});
+
+appInsights.loadAppInsights();
+appInsights.trackPageView(); // Manually call trackPageView to establish the current user/session/pageview
+
+console.log(process.env.NEXT_PUBLIC_APP_INSIGHTS_KEY)
```
Comment (oreHGA) [inline frontend/src/utils/appInsights.ts:23]: let's take this out

Comment (TemitopeAsama) [reply frontend/src/utils/appInsights.ts:23]: `appInsights.loadAppInsights();` too?

File: frontend/src/pages/auth/login.tsx (hunk 2 above)
Comment (TemitopeAsama) [reply frontend/src/pages/auth/login.tsx:53]: Done.

File: frontend/src/pages/profile.tsx (hunk 1 above)
Comment (TemitopeAsama) [reply frontend/src/pages/profile.tsx:18]: Done

File: frontend/src/utils/appInsights.ts (hunk 3 above)
Comment (oreHGA) [reply frontend/src/utils/appInsights.ts:23]: nope, just the log

**Pull request merged**